src/Error.js

```diff
@@ -8,7 +8,7 @@ class Error extends Component{
     return(
       <Alert variant="danger">
```

Comment:
Great idea implementing Alert for your error component. I used just an <h4> on mine and it takes more work to style it. I will definitely be refactoring it to utilize Alert as well.